### ISSUE TITLE
Typeahead's dropdown not always being announced to screen readers

### ISSUE THREAD

**Issue by emhaechi** [valor-software/ng2-bootstrap#5759]
**Bug description:**

If a user is using a screen reader (ex. NVDA, JAWS), when the typeahead's dropdown appears for the first time after a user types one character, the fact that a dropdown appeared and that an option is highlighted is not announced via the screen reader. However, if a second character is typed after that or the user navigates with the keyboard between the options in the list, then the screen reader announces the options as expected.

**Plunker/StackBlitz that reproduces the issue:**

https://stackblitz.com/edit/angular-etzfeq

**Versions of ngx-bootstrap, Angular, and Bootstrap:**

-  ngx-bootstrap: v.5.6.2
-  Angular: 9.1.0
-  Bootstrap: 4.4.1

**Expected behavior**

The dropdown list and highlighted option should always be announced to screen readers whenever they appears and change.

**Comment by apotapcukv** [comment]
Dear emhaechi!
As per our check this issue is fixed in the latest version, so we’re closing the issue.
Please don’t hesitate to let us know if you face further issues.
